app/controllers/api/base_controller/renderer.rb

```diff
@@ -549,12 +549,20 @@ def render_options(resource, data = {})
         render :json => OptionsSerializer.new(klass, data).serialize
       end
 
-      def render_update_resource_options(id)
+      def render_update_resource_options(id, action)
         type = @req.collection.to_sym
         resource = resource_search(id, type)
         raise BadRequestError, resource.unsupported_reason(:update) unless resource.supports?(:update)
 
-        render_options(type, :form_schema => resource.params_for_update)
+        if action == "attach"
```

Comment:
I don't think overriding the `render_update_resource_options` method is the way to go here

For example won't we not get here if we are looking for the "attach" action but the resource doesn't support "update" a totally different action? https://github.com/ManageIQ/manageiq-api/pull/1141/files#diff-48fb0a597ebc41e5c8c4fde22100d5577a08a07b84761152f69eaa2f580c4c37R555

Plus "update" is in the name, then we also have it handle other options